tests/lib/hardware/openstack_libcloud.py

```diff
@@ -201,6 +201,11 @@ def get_ssh_ip(self):
         # NOTE(jhesketh): For now, just use the last floating IP
         return self._floating_ips[-1].ip_address
 
+    def add_data_disk(self, capacity):
+        # TO BE IMPLEMENTED
+        logger.warn('add_data_disk() not implemented for'
+                    'openstack backend yet')
```

Comment:
(nit) We actually do add a data disk for openstack already.

This is fine for now, but the todo item is therefore to rework the existing methods into this interface.

Reply:
I updated the comment